### PR TITLE
Made custom list for trials, still need to implement.

### PR DESCRIPTION
**Pull request by edenemilynan**
**Altered trials to have setters instead of methods that increment the result or some variation of that. It might need to change back if I'm misunderstand how the doc-to-object business works, so I want to make note of it.**

### CODE REVIEW

File: project/app/src/main/java/com/example/cmput301w21t25/managers/TrialManager.java [hunk 1]
```diff
@@ -70,6 +76,8 @@ public void onFailure(@NonNull Exception e) {
                     }
                 });
     }
+  
+    public void FB_CreateBinomialTrial(String ownerID,String parentExperimentID,String parentExperimentName,String parentExperimentOwnerName, boolean published,boolean result,Experiment parent, Date date){
```
Comment (dkrammer) [inline project/app/src/main/java/com/example/cmput301w21t25/managers/TrialManager.java]: Move this to line 91, replace the method declaration there

File: project/app/src/main/java/com/example/cmput301w21t25/managers/TrialManager.java [hunk 2]
```diff
@@ -112,6 +121,8 @@ public void onFailure(@NonNull Exception e) {
                 });
     }
 
+    public void FB_CreateMeasurementTrial(String ownerID,String parentExperimentID,String parentExperimentName,String parentExperimentOwnerName, boolean published,float result,Experiment parent, Date date){
```
Comment (dkrammer) [inline project/app/src/main/java/com/example/cmput301w21t25/managers/TrialManager.java]: Same thing, replace line 136 with this one

**Review by dkrammer — CHANGES_REQUESTED** [review]
Just need to change a couple of the method declarations, looks like some merge conflict shenanigans

**Review by curtiskennedy — APPROVED** [review]
(my bad I missed Duncan's comment)

**Review by dkrammer — APPROVED** [review]
### NICE

**Pull request merged**